src/ContractType/contractType.js

```diff
@@ -0,0 +1,13 @@
+const ContractTypeTag = ({ contractType }) => {
+
+
+    return (
+        <>
+            {contractType !== null &&
```

Comment:
move the conditional rendering to surround the place that the component is used rather than inside it